### PR TITLE
chore(server): adjust daemon

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/lib/ckb_authentic_sync.rb [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-options = {
-  log_output: true,
-  log_dir: Rails.root.join("log"),
-  monitor: true,
-  dir: Rails.root.join("tmp", "pids").to_s
-}
-Daemons.run_proc("#{Rails.env}_ckb_authentic_sync", options) do
-  CkbSync::AuthenticSync.start
-end
+loop do
```
Comment (classicalliu) [inline server/lib/ckb_authentic_sync.rb]: Why not use rake tasks

Comment (shaojunda) [reply server/lib/ckb_authentic_sync.rb]: After using `system to  start daemons, I don't think it is necessary to use rake tasks.

**Comment by shaojunda** [comment]
Please don't merge, I need to check some details. @classicalliu

**Comment by shaojunda** [comment]
ready for review

**Comment by shaojunda** [comment]
ready for review @classicalliu

File: server/lib/ckb_authentic_sync.rb (hunk 1 above)
Comment (classicalliu) [reply server/lib/ckb_authentic_sync.rb]: rake task is a process in system, `systemd` to manage this process, this logic looks like a task and not belongs to main process.

**Pull request merged**